### ISSUE TITLE
Add a section for why Chrome/Chromium should not be used just like the Windows 10 section?

### ISSUE THREAD

**Issue by C-O-M-P-A-R-T-M-E-N-T-A-L-I-Z-A-T-I-O-N** [privacytoolsIO/privacytools.io#274]
Chrome/Chromium is the most used browser currently. Adding warnings and reasons against it may be a good idea.

**Comment by kewde** [comment]
The security of Chromium is very solid tho.
https://www.chromium.org/Home/chromium-security/brag-sheet

**Comment by C-O-M-P-A-R-T-M-E-N-T-A-L-I-Z-A-T-I-O-N** [comment]
@kewde

**Comment by C-O-M-P-A-R-T-M-E-N-T-A-L-I-Z-A-T-I-O-N** [comment]
@kewde 

> The security of Chromium is very solid tho.

But the privacy that it provides is very weak. This is similar to how Windows 10 is the most secure Windows OS but it is also the weakest when it comes to privacy.

**Comment by kewde** [comment]
@C-O-M-P-A-R-T-M-E-N-T-A-L-I-Z-A-T-I-O-N 
I know, there is a project 'chromium ungoogled', which was student project at the time but I believe some new developers have picked it up.

**Comment by C-O-M-P-A-R-T-M-E-N-T-A-L-I-Z-A-T-I-O-N** [comment]
@kewde 

> I know, there is a project 'chromium ungoogled', which was student project at the time but I believe some new developers have picked it up.

Removing the Google parts is, alas, insufficient to counter the fact that Chromium is very weak at fingerprinting resistance and first party isolation. Contrast this with how Firefox uplifts Tor Browser patches directly into Firefox (e.g. all it takes is to set `privacy.resistFingerprinting`, `privacy.firstparty.isolate` to `true` in `about:config`).

**Comment by kewde** [comment]
@C-O-M-P-A-R-T-M-E-N-T-A-L-I-Z-A-T-I-O-N 

Agreed.

**Comment by Shifterovich** [comment]
I'd recommend both ungoogled chromium and FF as some people may prioritize security over privacy, and even though we're a privacy website, we should focus on security as well.

This is individual, depends on what the user is looking for.

**Comment by C-O-M-P-A-R-T-M-E-N-T-A-L-I-Z-A-T-I-O-N** [comment]
> I'd recommend both ungoogled chromium and FF as some people may prioritize security over privacy, and even though we're a privacy website, we should focus on security as well.

There is no privacy with ungoogled chromium, all the stuff that you can easily fingerprint with Chromium (fonts, canvas, ...) is fairly sufficient, and when coupled with the lack of any first party isolation, shows that one can't expect to be safe from browser fingeprinting and tracking with it.

**Comment by Atavic** [comment]
I have a simple test for anyone on Windows who also has TCPView by [Sysinternals](https://technet.microsoft.com/en-us/) (Free):

- Browse any site with Firefox or Palemoon and look at TCPView after some pages load.

- Browse same sites with Chrome or Chromium and look at TCPView.

In the latter case _only_, I see svchost.exe connecting to Google IP in a permanent way between different sites. I can confirm it happens with Chrome and Chromium (haven't tried with [Ungoogled](https://github.com/Eloston/ungoogled-chromium) Chromium) as I have to allow svchost.exe on my firewall: something that doesn't happen with FF, Palemoon or similar browsers.

**Comment by Shifterovich** [comment]
No need to test the obvious. We're talking about Ungoogled Chromium.

BTW, one of the things Chrome sends to Google is the URLs you visit. This will of course happen with FF Sync as well.

**Comment by C-O-M-P-A-R-T-M-E-N-T-A-L-I-Z-A-T-I-O-N** [comment]
@Atavic 
That's detailed in this discussion, https://github.com/nylira/prism-break/issues/169

**Comment by C-O-M-P-A-R-T-M-E-N-T-A-L-I-Z-A-T-I-O-N** [comment]
I can also mention some more other problems with Ungoogled or Googled Chromium:

--------------------

The worst offender on this front is the use of the Microsoft Windows CryptoAPI for certificate validation, without any alternative. This bug means that certificate revocation checking and intermediate certificate retrieval happen outside of the browser's proxy settings, and is subject to alteration by the OEM and/or the enterprise administrator. Worse, beyond the Tor proxy issues, the use of this OS certificate validation API means that the OEM and enterprise also have a simple entry point for installing their own root certificates to enable transparent HTTPS man-in-the-middle, with full browser validation and no user consent or awareness.

All of this is not to mention the need for defenses against third party tracking and fingerprinting to prevent the linking of Tor activity to non-Tor usage, and which would also be useful for the wider non-Tor userbase.

https://blog.torproject.org/blog/isec-partners-conducts-tor-browser-hardening-study

-----------------------

**Comment by 2E0PGS** [comment]
I think "Google Chrome" should be avoided but "Chromium" is fine.

Chromium is even the base for many fast / stable and some privacy focused browsers e.g. Brave.

Electron uses Chromium engine. It's a good engine. Just avoid the Google builds and get de-googled ones: https://chromium.woolyss.com/

**Comment by davidtabernerom** [comment]
Not pretty sure Chromium is completely un-googled at all.
Someone with more info, please?

**Comment by 2E0PGS** [comment]
I was not talking about stock version of chromium. I clearly linked a custom build. Someone built from source. That means they can add or remove any google as they like since chromium's source code is open. Many things like electron are built on chromium engine / core code.

**Comment by Atavic** [comment]
Related: https://github.com/privacytoolsIO/privacytools.io/issues/380#issuecomment-353878470

**Comment by CHEF-KOCH** [comment]
Hello, 
I'm directly involved into the Chromium source code and development. The misleading information which are spread here are nothing but wrong without any evidence. It's also not to mention that Chrome nor Chromium collects more data than Firefox (default settings). The thing is that once you're logged in into Firefox (Mozilla) or Chrome (Google) you basically agree to their own terms which means they can see what you sync etc. However, keep in mind that they won't see the real content because it's [encrypted](https://support.google.com/chrome/answer/165139?hl=en&co=GENIE.Platform%3DAndroid). Both, Mozilla and Google have only limited access to such data. The decryption of e.g. your passwords you sync to them needs you approval, you need to type in your password and you can setup additional separate passphrases which you can keep local (only you know this). This is also a bit more detailed explained over [here](https://superuser.com/questions/753465/im-using-chrome-and-chrome-sync-does-google-have-access-to-my-passwords). A proper implementation of [PBKDF2 is planned](https://bugs.chromium.org/p/chromium/issues/detail?id=820976). Mozilla also has a similar ticket for this [here](https://bugzilla.mozilla.org/show_bug.cgi?id=1320222).

About connections google or chrome makes:
* Most connections are loopback and mdns based checks which are 'needed' to identify if you're behind something, basically these are capital login checks (also implemented in Android, Firefox, iOS and basically every modern OS and Browser).
* Chromium doesn't communicate over svchost.exe. This process is DNS related and Windows own DNS cache among several other things, the connection you have to look at are the real processes which establish the real connection. 
* Ungoogled Chromium should not be used it's mostly behind or outdated of known forks or even official releases, it's questionable if that has a real privacy benefit since there is no evidence given. Together with the fact that you can (same like Firefox) 'improve' or 'harden' it's configuration which might already answer a lot of questions because you might look deeper into it's given options and how they work. Removing several things like ungoogled-chromium does is in most cases not needed since the possible connections are based on how you set your Browser up, which options you choose etc. Firefox and Chrome and every other Browser does connect straight from the start with default settings to several domains, to update the disconnect filter list or for the safebrowsing mechanism etc. Coming to a conclusion without providing evidence that this is spying is wrong, it's by default how the Browser implements their security mechanism.
* Firefox and Chrome/Chromium have their privacy-related things well documented. 
* If you want no connections you can start with my Chromium hardening project which shows and explains already a lot options which you can enable in order to reduce possible attack surface - it's similar to about:config related toggles like in Firefox.

![greg](https://user-images.githubusercontent.com/727123/40522506-8bce1e58-5fd1-11e8-8de9-35a40f5ad984.png)

As you can see I did a lot of effort, I even created a tunnel and used another router in topü of it so nothing can be hidden. It was tested in a VM with everything as default. 


If there a privacy-related question regarding Chromium feel free to contact me directly via my given contact channels and I might answer your privacy concerns. I can't question or show everything here because some people might get upset if you post such a long comment but I always try to give as much information as I can. 

I think the main thing is that the project doesn't explain much, it shows some tools, Browser etc but it doesn't explain much how you setup stuff in order to gain the maximum 'security' (whatever that really means). And this is why people still getting misleading information like that any Browser would spy on you, simply don't login into something that you don't like and use their services/search engines and then there should theoretically no connections at all unless there is explicity data collection which can't be opted-out (which applies for every Browser which might collect additional 'telemetry').

**Comment by kewde** [comment]
@CHEF-KOCH 

This is exactly the kind of research I appreciate - thank you for clearing up the svchost issue.
It seems like the author didn't include the link to his hardening project, so [I'll link it here](https://github.com/CHEF-KOCH/Chromium-hardening).

I think the general rule of thumb is: if you plan on using Tor - then go with the Tor Browser and don't use Chromium as certification retrieval (on Windows) is unproxied. I'm not sure if the same applies for VPN - I would _assume_ that a VPN will route connections of svchost through the VPN. 
However, on platforms such as Linux and perhaps even Mac, I think given the right custom setup, you can achieve comparable levels of privacy and security. 

I haven't seen any indication that chromium has more telemetry by default than Firefox.

What I would like to find out is how the different browsers (default & hardened) compare against fingerprinting. There's not "a" metric for fingerprinting, but we can use a variety of tools and compare results within each tool.

As far as security goes, I think Chromium wins hands down. I did hear Firefox recently landed namespace sandbox support (Linux) but it's still disabled or not compiled-in for quite a operating systems.

> I think the main thing is that the project doesn't explain much, it shows some tools, Browser etc but it doesn't explain much how you setup stuff in order to gain the maximum 'security' (whatever that really means).

That's true. The purpose of this website is to provide a simple entry level to a more private life on the internet. It's aimed at providing solutions that do no require custom configurations (often prone to errors). 

The maintainers of projects like this one, where we try to cover a large and broad range of applications, are at the mercy of their community. This project is only as smart as its community. As a maintainer my job is to analyze the evidence and supplemental resources that are provided by the community.
I know my set of skills and I know that compared to the broad set of tools we're covering, many areas of my knowledge are lacking hence I rely on having people smarter than me around.

I attempt to be a judge - but again, in threads like these when there is a single narrative pushed like 'google is evil', I will resort to doing my own research rather than verifying the research provided to me by you guys.

**Comment by CHEF-KOCH** [comment]
[The Answer why Chome communicates behind your back without user consent](https://chefkochblog.wordpress.com/2018/04/05/the-answer-why-chome-communicates-behind-your-back-without-user-consent/)

**Comment by csagan5** [comment]
@CHEF-KOCH you cannot disable all integrations from a vanilla Chrome/Chromium, even when using policies (which a regular user would not know how to accomplish anyways).

I agree that there is a lot of misinformation and a connection or data transfer to a Google server does not mean automatically that there is active spying going on (it never is a form of spying anyways if the user has accepted a contract). However let's stick to facts, shall we? Conflating the hyperbole "X is spying on me through these connections!!" to "there are no privacy concerns here" is flatly absurd and a FUD strategy by itself.

Facts (as I reckon):
* a connection to a server will already provide connection metadata (originating IP address, timestamp), which the user has the right to choose to not make it happen
* there are no easy to access options to disable each and every connection happening (imagine that you could group them and disable such groups of integrations)
* it is a hard sell to ask an user to trade an always up-to-date browser vs the previously mentioned privacy violations; user should be allowed to have an always up-to-date browser **and** the choice about which connections are happening and which data is being transferred.

We have nowadays this project (privacytools.io) and the projects you mentioned (of different quality and effectiveness) because of the failure (lack of interest/motivation/pressure in doing so?) at giving users (of any level of expertise) an easy choice on all the server-side integrations.

Since you mentioned that you are involved in Chromium development, let me ask this direct question: is there a Privacy cross-team panel that evaluates privacy concerns from the user perspective? I assume that you have a similar cross-team group of people for the security aspect of the browser, which is massively important.

If there is, can you elaborate on how big is the privacy evaluation group compared for example to some other group that works on the UI or anything else, and if it has importance and effectiveness in its reviews/vetoes?

This would tell a lot about the dynamics of privacy assessment in the development of the Chromium browser, but you probably cannot comment on this (and I understand); I hope you understand why I am asking this in first place.

In short: your project (chromium hardening), is it about security hardening or increasing privacy? Why is there a list of Firefox privacy scandals? I am not a fan or Firefox or anything, I am just asking why is that on-topic for a security hardening project, as I would have expected links to CVE rather than media coverage (all that media coverage to "disprove" and put other projects in a bad light makes it look like a gaslighting operation).

**Comment by blacklight447-ptio** [comment]
because the usage of the chromium engine does not in itself means that its privacy unfriendly, means that it should not be warned against like windows 10, which unlike chromium, IS always privacy unfriendly. closing issue.

**Comment by CHEF-KOCH** [comment]
Let me say that I'm not really anymore interested in privacytools.io and that I don't care what this project lists because a lot of the listed programs and suggestions are based on how popular the products are. Some listed projects/tools are also questionable and I gave up arguing with "fans" because at the end it's a waste of time, and they (in 99% of all cases can't even provide any source code or evidence for their claims). If you want evidence just wait for an audit (which ain't cheap).

There is also a constant "bashing" against product xyz because people might have bad experience with it even if the product/tool is perfectly fine. Among misinformation, false advertising and bullying to make product xyz look worse than others. This can be strategy, e.g. on Amazon such "fake reviews" are written by (some) paid people to make other products look worse than they really are. In order to push their own products. If you check the Wikipedia article you will notice that Chrome gets more bashing for years old mistakes and the Firefox article doesn't even list a criticism section - how objective is that?! This is clearly written from people which want to push Firefox or tool X. It's not like that the mentioned things are already removed. It's worth nothing listing old problems/mistakes/holes without mention which ones are actually open/closed and from interest - because why should an average Joe care about already fixed issues? 

- Both, Chrome and Firefox evolved and both learned from past mistakes - this is normal and okay.
- There are privacy friendly alternatives for both, Chrome/Chromium and Mozilla Firefox. Pre-made packages which are slimmed or tweaked down for maximum privacy which of course will cripple the web experience at some point. 
- I found NO Browser which is truly privacy friendly by default because that will cripple most "fun experience" on the web including, nope not even the all mighty Tor Browser (Firefox based).
- Privacy depends on several facts, like what you use/need and what the Browser offers and if there are opt-in options given via "Settings" or not however in both Firefox and Chrome a lot (not all) possibilities are not visible for the user, you have to go to all the about:config & about:flags stuff in order to make it "[privacy friendly](https://www.chromium.org/Home/chromium-privacy)"
- Privacy friendly is a marketing term created in order to make profit or to hype product x - Because it's "better" (without any proof just marketing phrases & other things). 
- There is the usability aspect which is never been mentioned, why disallow e.g. WebGL when page xyz like e.g. ShaderToys really needs it? The website is legit and does not spy just because you enable WebGL, if there are trackers then there are anyway implemented in a way to track you with WebGL enabled or not, you change at the end nothing except the possibility to get "less tracked". The problem isn't the Browser, it's the webmaster which decided to implemented tracker xyz in the first place. This abstract example can be verified by Adobe Flash vs. HTML5, now HTML5 removed it and has its own flaws, so you changed nothing except lowering the attack surface which does not mean there are no attack scenarios for HTML5 too. It's naive to think that if you change a product or protocol the old flaws magically disappearing. 
- Listing CVE's is controversial because (again) why should someone care about the ones which are already addressed and keeping an eye on every single CVE is a lot of work. Maintaining such CVE databases like Mitre etc. are jobs for a lot of people or entire groups. It's also unclear if most people understand the source code or understand how to interpret the CVE. However, the "important" ones, aka critical ones are mostly if not always mentioned in the Firefox & Chrome changelog (which no one reads anyway - not a normal person or the IT admin which updates tons of PC's with the same Browsers installed.

I do not understand why people constantly saying Chromium is bad, there is no hidden spying implemented and as said you can use alternatives like e.g. Brave if you want the "best" out-of-the-box experience. Same goes for Firefox, the default version is nothing compared to "tweaked" ones you can grab here on GitHub when it comes to privacy.


The privacy aspect was and still is (I can say this out loud) in the Chrome/Mozilla team a big aspect and if you check both issue trackers you find the tags "privacy" & "security" related among a lot of other tags for those questions/issue tickets. Yes, [there are working a lot of people behind](https://www.chromium.org/Home/chromium-security) in order to make the Browser "secure". This is proven almost every year again on Blackhat conference or the Pwn2Own contest which often reveals critical flaws in Firefox/Chrome etc (which then immediately getting fixes after they got leaked - if you watch the Pwn2Own history you will see Firefox is more often "pwned" than Chrome/Chromium because the Sandbox (to mention one aspect) was and still is a lot of better in Chrome/Chromium. Most if not all "flaws" often requiring physical access to the PC or a manipulated Windows to "hack" the Browser. In my point of few this is not even a CVE then it's more a POC and should be (if possible) fixed within the OS to prevent drive-by in the first place (in a lot of cases this is not even possible and often gets closed as "wontfix" because [it's not Browser relevant)](https://chromium.googlesource.com/chromium/src/+/master/docs/security/faq.md).


I think the question about how many people care/working on the privacy & security aspect says nothing about the quality, the product still could be flawed because a Browser is a "never finished" product and new stuff gets added while other parts getting removed, this will always cause "problems" and no software I've seen is perfect. 


My impression is that Reddit & Co. spreading the "pro Firefox" thingy since years without any real proof, so everyone is just re-spelling it without checking the source code - and the source code is good enough so that I can say "there is nothing wrong with Chromium". Keep in mind that the connection lots of folks complaining about is not even Browser related, lots of stuff a Browser includes are from third-party libraries and sub-functions like e.g. the PDF reader, the Chromecast function and and and that you can't even say "it's the browser fault" because during the merging & reviewing process  you might adopt flaws from those mentioned sub-projects.

Don't get me wrong, I even would use Internet Explorer if it works "best" for me or if it would be more secure/privacy friendly. I usually decide to go (which I think) is the "best" (in my eyes) and I don't care if it's Firefox/Chrome/Safari or any of the mentioned products listed here.


So why did I wrote this if I don't care, because I was mentioned and to get out of the "Chrome/Chromium" is evil loop. If there is any evidence then someone would have already revealed it and no one did. The open source argumentation is also irrelevant because you can also reverse any other binaries (if you want to) with a lot of tools to inspect what is really going on and the rest can be captured with a reverse proxy to see what the connection really transmit, in a lot of cases it's just loopback to check e.g. if there are Chromecast, a USB-drive etc connected to the Browser (most of this - I agree not all of it) can be disabled in Chrome/Chromium vanilla same like in Firefox vanilla via about:flags/about:config. I do agree that this is controversial because in my opinion this should be possible via "Settings" so that every average Joe can config or pre-config (via profiles) their Browser how he/she wants to and now what Mozilla/Google "dictates" as the default (because most people still using defaults as the daily standard), sometimes it's even blocked by an Admin/Policy setting to prevent altering those flags/settings. 

And no, Tor is not the solution in most "secure" environments I ever worked the Admin tried to block the entire Tor network/connections and the Browser itself in order to prevent people from bypassing xyz site restriction (porn, etc.). Which is imho okay because at work you get paid for work and not to watch such crap.

**Comment by blacklight447-ptio** [comment]
first of all, we are about to list one chromium based browser because of the security benefits on android, and are still searching for an desktop one. 

Second of all, which products do you think are listed because of popularity.

And as third, if you were in our shoes, what would you recommend ;).

**Comment by CHEF-KOCH** [comment]
On the Desktop level, I would suggest:
- In general list tools/websites in alphabetically order to avoid the "so is xyz better than zsg because it's listed first?" questions)
- List Chromium
- List Firefox
- Maybe other Browser ("original ones") (to be honest in my above statement I did not mentioned/included any mobile OS based Brwser for a reason). However, in most cases (as of today it's Chrome/Chromium Vs. Firefox anyway -> maybe Edge [Chromium based]).
- Create a sub-category with "hardened" or tweaked (or whatever you like to call it) Browsers and list the ones which are well known like Brave etc. and show/link to the website/FAQ which list their benefit(s). The privacytools.io website already lists (some) about:config tweaks, I maybe would add this to the original Browser section and explain why such tweaks are interesting. I also would mention some Chrome/Chromium flags.

Last time I checked it the privacytools.io website listed the mobile category separated (which makes sense), I would suggest using the same strategy as mentioned above, simply list the "known secure/privacy friendly" alternatives and then add alternatives like e.g. for Android "privacy Browser" (Firefox based), Brave etc. 

Instead of "bashing" Chrome/Chromium/Firefox I highly suggest to list the benefits using "original" (non-tweaked Browsers), this would be the fact that usually more people working on such projects and that they regularly getting updates etc. And then I would say list the modified/tweaked/hardened ones and list their benefits, e.g. that they are pre-configured etc.

I def. would not mention that Chrome/Chromium/Firefox had some issues or privacy concerns (because no Browser is or was perfect) instead I would focus on the things which are important and list them e.g why should user xyz use/prefer browser x. As mentioned the tweaked/hardened Browsers are usually the ones which are easier for beginners but often (not always) lacking of updates (because the community or developer team is smaller or have to adopt the changes from the original Browsers first).

I explicitly did not mention any Browser names/alternatives above because the topic was/is Chrome/Chromium and there already lots of opened/closed issue tickets about possible alternatives.

I'm highly against a "should not be used" category because there has to be evidence to support such claims and outdated links/information is nothing but confusing and leads to misunderstandings. Windows 10 constantly changes so does Chrome/Chromium/Firefox. I'm more for the idea that you only list the things the user can do in order to avoid e.g. tracking rather than constantly repeat xyz is evil without anything. Controversial or not, it's not helpful crying all day long without giving your visitors any solutions and I think the website has already a huge influence, which means that you should stay as neutral as possible. 

For example a lot of people asking me how to remove the Windows packages (fully), they usually don't care if Windows submits telemetry or not because there are "no alternatives" (to them) instead they want the information how-to remove xyz and I think that's what privacytools.io should do instead.

**Comment by blacklight447-ptio** [comment]
I think you would like to see our current PR to improve the browser section, we list chromium based alternative bromite for android there, and firefox is a recommendation across all three platforms:https://github.com/privacytoolsIO/privacytools.io/pull/1256

The project now has a pretty stable team who constantly works on it, me included. So things will be a lot less about whats more popular, and more about actual facts.we are now also redoing most of the webpage,  re-investigating listed software, and adding a clear minimum criteria, you can take a look at the vpn section, which was recently redone.

In any case, the project has grown more mature now. so past things you saw that were annoying, should be much less of a problem nowadays.

**Comment by csagan5** [comment]
First: I am sorry if you have been attacked on the internet for your opinions, everyone should (ideally) respect each other's opinions and be polite online. I felt compelled to write because I saw in your post some inexact statements and some of the techniques you criticise. But there is a simple way to dispel FUD (claimed or real): fact checking.

> * Privacy depends on several facts, like what you use/need and what the Browser offers and if there are opt-in options given via "Settings" or not however in both Firefox and Chrome a lot (not all) possibilities are not visible for the user, you have to go to all the about:config & about:flags stuff in order to make it "[privacy friendly](https://www.chromium.org/Home/chromium-privacy)"

> in a lot of cases it's just loopback to check e.g. if there are Chromecast, a USB-drive etc connected to the Browser (most of this - I agree not all of it) can be disabled in Chrome/Chromium vanilla same like in Firefox vanilla via about:flags/about:config.

I am glad you agree that not all interactions with services can be disabled, because it is exactly what a lot of developers have figured out already. To give another little evidence: Chromium requires users to agree to ToS before starting using it because a lot of data transmission (think about metrics on the usage of various features, experiments etc) cannot be opted out. Thus, by law, the browser must ask user permission and/or it says that by starting to use it the user automatically agrees to service usage conditions.

One can argue till the end of time that there is no [PII](https://csrc.nist.gov/glossary/term/personally-identifiable-information) in such data, it does not matter: modern privacy legislation requires consent for almost any type of information exchange and that is exactly why user must accept (explicitly second GDPR, implicitly in most other countries) a service agreement.

> I do agree that this is controversial because in my opinion this should be possible via "Settings" so that every average Joe can config or pre-config (via profiles) their Browser how he/she wants to and now what Mozilla/Google "dictates" as the default (because most people still using defaults as the daily standard)

It is IMO nothing else that a [dark pattern](https://www.darkpatterns.org), on a larger scale.

> * Listing CVE's is controversial

I proposed this versus listing opinion pieces. My critique/suggestion is that at least CVEs are recorded facts. Opinions do not go through the same criteria of acceptance as a CVE. I see you disagree with some CVEs; fine, they are still better than average Joe's opinion expressed in a blog post.

> Yes, [there are working a lot of people behind](https://www.chromium.org/Home/chromium-security) in order to make the Browser "secure".

That is not what I asked. I asked whether they have steering power to protect user privacy. Security, it's a different beast. Treating the two altogether is damaging for both the privacy and security aspects (although [privacy violations have begun being addressed as security violations](https://webkit.org/tracking-prevention-policy/)).

> My impression is that Reddit & Co. spreading the "pro Firefox" thingy since years without any real proof

It's not just that, Firefox markets itself to be better for this reason (as Brave does). Competing browsers do not have any other choice than to show that they are better in what the dominating browser is **perceived** to be worse. They locate the weakest aspect of the dominating browser and try to win there. Now, one could say that such user perception is completely manoeuvred by marketing and propaganda. I politely disagree; once again, the history of browser changes (in features) can tell us more than 1000 blog posts.

I agree with other parts of your posts, not cited here, for example regarding the more general role of this project and how to be of service to users.
